frontend/src/Search.js

```diff
@@ -309,6 +309,7 @@ class Search extends Component {
     return (
         <div>
         <input type = "text" 
+          name="search-text"
```

Comment:
Don't use `name`, that's only used in forms. If you want to identify an element, use the field `id`